update docker build command to support arm64

niradler opened the pull request:
Hi, I'm running postgres on cloud instance base on arm architecture, works great but when trying to use the postgis image got issue related to missing arm version

phillipross commented:
@niradler thanks for eagerness to contribute.

I'm going to approve you for the purposes of running the CI workflows for your PRs, but quick look at this PR seems that you're change is wholesale switch from amd64 to arm64 platform which is definitely not what we want.  We'll actually want to build for as multiple platforms.  I'll let the workflow run, though, so you can see how far through the CI workflow your changes get.  Thanks again!

niradler commented:
Hi, Thanks, updated the pr to keep original build and just added another line for arm support, this way no breaking change, but eventually, the cleaner way will be to build all in one command with buildx
